examples/a_treeple/treeple_tutorial_CMI.py

```diff
@@ -0,0 +1,149 @@
+"""
+====================================
+Treeple tutorial for calculating CMI
+====================================
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.stats import entropy
+
+from sktree.datasets import make_trunk_classification
+from sktree.ensemble import HonestForestClassifier
+from sktree.stats import build_hyppo_oob_forest
+from sktree.tree import MultiViewDecisionTreeClassifier
+
+# %%
+# CMI
```

Comment:
I would spell out CMI first. Not everyone knows what CMI stands for. In addition, perhaps it is useful to specify in words what CMI will tell us.